src/components/Tooltip/Tooltip.tsx

```diff
@@ -83,12 +95,19 @@ export const Tooltip = ({
   align = "center",
   onEscapeKeyDown,
   onPointerDownOutside,
+  isTriggerInteractive = true,
   open,
 }: PropsWithChildren<TooltipProps>): JSX.Element => {
   return (
     <Provider>
-      <Root open={open} delayDuration={300}>
-        <Trigger asChild>{children}</Trigger>
+      <Root open={open} delayDuration={isTriggerInteractive ? 300 : 0}>
+        <Trigger asChild>
+          {isTriggerInteractive ? (
+            children
+          ) : (
+            <span tabIndex={0}>{children}</span>
```

Comment:
This will end up problematic for more complex widgets which want roving tab indexes to avoid thousands of tabstops in a given page.